packages/api-kit/tests/e2e/addSafeOperation.test.ts

```diff
@@ -1,15 +1,19 @@
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
-import { ethers } from 'ethers'
 import sinon from 'sinon'
 import sinonChai from 'sinon-chai'
 import Safe from '@safe-global/protocol-kit'
 import SafeApiKit from '@safe-global/api-kit/index'
-import { Safe4337Pack } from '@safe-global/relay-kit'
+import { getAddSafeOperationProps } from '@safe-global/api-kit/utils/safeOperation'
+import { BundlerClient, Safe4337Pack } from '@safe-global/relay-kit'
 import { generateTransferCallData } from '@safe-global/relay-kit/packs/safe-4337/testing-utils/helpers'
-import { RPC_4337_CALLS } from '@safe-global/relay-kit/packs/safe-4337/constants'
+import {
+  ENTRYPOINT_ABI,
+  ENTRYPOINT_ADDRESS_V06,
+  RPC_4337_CALLS
+} from '@safe-global/relay-kit/packs/safe-4337/constants'
+import * as safe4337Utils from '@safe-global/relay-kit/dist/src/packs/safe-4337/utils'
```

Comment:
We should not import from dist folder

Reply:
I had to import it like this in order to mock the `getEip4337BundlerProvider` function (see line 44). As this is a different package, the mock does only work when the dist version is used here.